resources/js/Components/Galleries/NftGalleryCard.blocks.tsx

```diff
@@ -184,6 +212,8 @@ export const NftImageGrid = ({
                         }}
                         isSelected={isSelected}
                         isAdded={isAdded}
+                        combinedNfts={combinedNfts}
+                        nftLimit={nftLimit}
```

Comment:
Looks fine, you could improve this further: instead of passing `combinedNfts` and `nftLimit`  a `reachedLimit: boolean` prop would also work, currently you are passing limit and selected NFTs to do calculation in the `NftImageContainer`, however, a simple flag would make things simpler.